test/test-lib/fixtures.ts

```diff
@@ -15,6 +15,41 @@ type PendingFixtures = Dictionary<PromiseLike<Dictionary<PromiseLike<any>>>>;
 type PartiallyAppliedFixtures = Dictionary<Dictionary<PromiseLike<any>>>;
 export type Fixtures = Dictionary<Dictionary<any>>;
 
+interface AuthUser {
+	id: number;
+	token: string;
+}
+
+interface ApiRequestOpts {
+	url: string;
+	headers: AnyObject;
+	method: string;
+	json: boolean;
+	body: AnyObject;
+}
+
+const buildApiRequestOpts = (args: {
+	path: string;
+	method?: string;
+	body?: AnyObject;
+	user?: AuthUser;
+}): ApiRequestOpts => {
+	const { path: reqPath, method = 'POST', body = {}, user } = args;
+	const headers: AnyObject = { 'X-Forwarded-Proto': 'https' };
+
+	if (user != null) {
+		headers.Authorization = `Bearer ${user.token}`;
+	}
+
+	return {
+		url: `http://${API_HOST}/${reqPath}`,
+		headers,
+		method,
+		json: true,
+		body,
+	};
+};
+
```

Comment:
This is essentially a duplicate of the `createResource` function, can you use that instead please

Reply:
now using `createResource` instead, removed those declarations.